machine_learning/dbscan/dbscan.ipynb

```diff
@@ -0,0 +1,318 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## DBSCAN"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Requirements:__\n",
+    "\n",
+    "Sadly, we have some unavoidable requirements.\n",
+    "1. Matplotlib for visualization\n",
+    "2. Scikit-learn for grabbing some standard datasets to test on\n",
+    "3. Numpy\n",
```

Comment:
Good news.  https://github.com/TheAlgorithms/Python/blob/master/requirements.txt already loads all thos dependencies (and many more).

Reply:
Alright. I should have noticed that.